From 9c4dbed56078d79022e312de8754393773415e2a Mon Sep 17 00:00:00 2001
From: Drew <drew@localhost>
Date: Sat, 25 Jul 2020 11:30:24 +0200
Subject: [PATCH] fix `inkscape` command line
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add `dbus-run-session` to avoid all warnings about D-Bus and bump command
line options to v. 1.

Signed-off-by: Drew <drew@localhost>
Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index 9d80f46..b1e9a81 100755
--- a/build.sh
+++ b/build.sh
@@ -109,7 +109,7 @@ function render {
 
   mkdir -p "$BUILD_DIR/$variant/$name"
   find "$SRC/svg/$variant" -name "*.svg" -type f \
-      -exec sh -c 'inkscape -z -e "$1/$2/$3/$(basename ${0%.svg}).png" -w $4 -h $4 -d $5 $0' {} "$BUILD_DIR" "$variant" "$name" "$size" "$dpi" \;
+      -exec sh -c 'dbus-run-session inkscape -o "$1/$2/$3/$(basename ${0%.svg}).png" -w $4 -h $4 -d $5 $0' {} "$BUILD_DIR" "$variant" "$name" "$size" "$dpi" \;
 }
 
 # Assembles rendered PNGs into a cursor distribution.
-- 
2.26.2

